commands/describe.go

```diff
@@ -146,19 +146,23 @@ func printFunctionDescription(funcDesc schema.FunctionDescription) {
 	fmt.Fprintln(w, "Function process:\t "+funcDesc.EnvProcess)
 	fmt.Fprintln(w, "URL:\t "+funcDesc.URL)
 	fmt.Fprintln(w, "Async URL:\t "+funcDesc.AsyncURL)
+	printLabels(w, "Labels", *funcDesc.Labels)
+	printLabels(w, "Annotations", *funcDesc.Annotations)
 
-	if funcDesc.Labels != nil {
-		fmt.Fprintf(w, "Labels:")
-		for key, value := range *funcDesc.Labels {
-			fmt.Fprintln(w, " \t "+key+" : "+value)
-		}
+	w.Flush()
+}
+
+func printLabels(w *tabwriter.Writer, name string, labels map[string]string) {
```

Comment:
On first glance, it looks like you forgot about annotations, because the name is still "labels". Perhaps the name of the generic function needs to change to printMap?

Reply:
When you make this change, run `git commit --amend -s` so that you don't create a PR with two commits, but just a single commit that's the final desired state.

Reply:
I had a look at the kubectl repo and they followed the same pattern, using printLabels for annotations as well. I'll amend it to something more intuitive.